scio-examples/src/test/scala/com/spotify/scio/examples/extra/JavaConvertersExampleTest

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright 2018 Spotify AB.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.spotify.scio.examples.extra
+
+import com.spotify.scio.testing.{CustomIO, PipelineSpec, TextIO}
+
+class JavaConvertersExampleTest extends PipelineSpec {
+
+  val output = "gs://bucket/path/to/file"
+  val expected: List[String] = (1 to 10).toList.map(_.toString)
+  val io = CustomIO[String](output)
```

Comment:
Since the pipeline uses the TextIO transform, it seemed consistent to expect it to return an `SCollection[String]` in the text. Since `.saveAsCustomOutput` doesn't actually apply the transform in testing, though, we have to manually map Int -> String in the pipeline.